process_win.cpp

```diff
@@ -5,9 +5,35 @@
 
 Process::Data::Data(): id(0), handle(NULL) {}
 
+namespace {
```

Comment:
Yes, it makes the `Handle` class visible only in this file (which means that we can define another `Handle` class in other files, e.g. in case the user has a `Handle` class in his/her program). It's like defining a variable `static`, but in this case is a type. (If you compile two .cpp files with a `class Handle` defined and implemented on them, you will have problems in the linking process).